### DISCUSSION TITLE
Interacting with Individual Components of a Cross Section

### DISCUSSION THREAD

**Discussion by WOHughes** [idea-statica/ideastatica-public#686] [Rhino/Grasshopper] [answered]
Hello,

I am attempting to model a stiffener inside a member with a component cross section. Previously I have achieved this by manually cutting the stiffener plate with each component of a cross section, one by one. However, I am now attempting a similar task using the Grasshopper plug-in, and am struggling to find a way of doing this. 
Is it possible to cut a plate with a specific component of a member with a component cross section in GH?

What is exported to Statica:
![image](https://github.com/user-attachments/assets/444d602a-21b3-4dd3-81cb-fe4ff9e89580)
![image](https://github.com/user-attachments/assets/2b25b260-11d0-425a-8539-5aaf5d566958)

What is desired:
![image](https://github.com/user-attachments/assets/b7bb3907-1bc3-4baf-9333-53a08b2ec992)
![image](https://github.com/user-attachments/assets/4849a42a-aebc-453c-b2bc-e886ac57b3ea)

Is there any way to supply in these three components as the cutting members?
![image](https://github.com/user-attachments/assets/9c2ee927-7e08-4046-af6d-ded99dd79f37)
![image](https://github.com/user-attachments/assets/f31c27e2-9366-497a-a3de-384fde58f128)

If this is not yet achievable using the GH plug-in, would I be better off using IOM directly? Thanks!

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi @WOHughes,
I have replicated your issue. It appears in this instance the setting of the +/- cut area of the plate does not appear to work correctly. I need to touch base internally to see if this can be fixed.  
![image](https://github.com/user-attachments/assets/f18d926a-5f05-4511-ac0e-030ce08dc963)
![image](https://github.com/user-attachments/assets/3e1965f3-1bc3-4140-a265-879acdf3c987)
In the meantime. Seeing though you have modelled your cross-section so precisely you could model the plate and welds explicitly. I did a quick test and seems to work okay. For some reason the second weld does not get imported, but we will fix this. Example Files below.
![image](https://github.com/user-attachments/assets/f664c38d-d3ec-4b24-8989-d782f014c0ed)
![image](https://github.com/user-attachments/assets/1167ec2d-6295-401a-acc6-fafb58cf9b3f)
 
[CutTest.zip](https://github.com/user-attachments/files/17468362/CutTest.zip)
